third_party/chromium/media/base/starboard_utils.cc

```diff
@@ -396,6 +399,24 @@ SbMediaColorMetadata MediaToSbMediaColorMetadata(
 
   return sb_media_color_metadata;
 }
+
+int GetSbMediaVideoBufferBudgetMaximum() {
+  LOG(INFO) << "Checking for maximum video buffer budget.";
+  auto budget = SbMediaGetVideoBufferBudget(SbMediaVideoCodec::kSbMediaVideoCodecH264, 1920, 1080, 8);
+  if (SbMediaCanPlayMimeAndKeySystem(kVp9_8k_mime, "") == kSbMediaSupportTypeProbably) {
+    budget = SbMediaGetVideoBufferBudget(SbMediaVideoCodec::kSbMediaVideoCodecVp9, 7680, 4320, 12);
+    LOG(INFO) << "Device supports 8k.";
+  }
+  else if (SbMediaCanPlayMimeAndKeySystem(kVp9_4k_mime, "") == kSbMediaSupportTypeProbably) {
+    LOG(INFO) << "Device supports 4k.";
+    budget = SbMediaGetVideoBufferBudget(SbMediaVideoCodec::kSbMediaVideoCodecVp9, 3840, 2160, 12);
```

Comment:
We should avoid calling any Sb functions inside Chromium media, consider using `DecoderBuffer::Allocator::GetInstance()->GetVideoBufferBudget()` as below.

Reply:
removed.